src/sympc/protocol/fss/fss.py

```diff
@@ -180,6 +180,8 @@ def fss_op(x1: MPCTensor, x2: MPCTensor, op="eq") -> MPCTensor:
 class FSS(metaclass=Protocol):
     """Function Secret Sharing."""
 
+    tensor_class = ShareTensor
```

Comment:
This should be changed to share_class

Reply:
Also, why is share_class not an attribute of FSS?

Reply:
What do you mean by attribute?

Reply:
Ahh, never mind. I got the answer to my question. I just realized protocol functions are static.